python/tchannel/rw.py

```diff
@@ -35,7 +35,11 @@ def number(width_bytes):
     return NumberReadWriter(width_bytes)
 
 
-def len_prefixed_string(length_rw, is_binary=False):
+def args(length_rw, is_binary=True):
+    return ArgsReaderWriter(length_rw, is_binary=True)
```

Comment:
IMO you don't need `is_binary` to be customizable for `args`. It should always be True.